server.ts

```diff
@@ -324,7 +324,7 @@ function initCache() {
     botCache = new LRU( {
       max: environment.cache.serverSide.botCache.max,
       ttl: environment.cache.serverSide.botCache.timeToLive || 24 * 60 * 60 * 1000, // 1 day
-      allowStale: true // If object is found to be stale, return stale value before deleting
+      allowStale: environment.cache.serverSide.botCache.allowStale || true // if object is stale, return stale value before deleting
```

Comment:
Using the `||` operator here makes it so `allowStale` can never be turned off through configuration.
It should be a [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) instead.

Reply:
@ybnd : Good catch!  I completely missed that ... and this was definitely a copy & paste error.  Just pushed up a fix a moment ago in https://github.com/DSpace/dspace-angular/pull/2033/commits/9e11d69a8ade6fae07676d49f6a2cb4697705107